atd-vze/src/views/Locations/Location.js

```diff
@@ -130,6 +194,66 @@ function Location(props) {
       )}
       <Row>
         <Col lg={6}>
+          <Card>
+            <CardHeader>Types of Vehicles - Count Distribution</CardHeader>
+            <CardBody>
+              {data.atd_txdot_crashes_aggregate.aggregate.count === 0 && (
```

Comment:
we use `data.atd_txdot_crashes_aggregate.aggregate.count` at least 4 times here. Before the `return (` on 138, why don't we do

```js
const { count: crashCount } = data.atd_txdot_crashes_aggregate.aggregate
```

so you can use a more human friendly variable `crashCount` instead.